src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -149,7 +148,7 @@ public void testBQSRSpark(BQSRTest params) throws IOException {
     public Object[][] createBQSRCloudTestData() {
         final String localResources =  getResourceDir();
 
-        final String GRCh37RefCloud = ReferenceAPISource.URL_PREFIX + ReferenceAPISource.GRCH37_REF_ID;
+        final String GRCh37RefCloud = GCS_b37_CHR20_21_REFERENCE;
```

Comment:
Couldn't you remove the `GRCh37RefCloud` variable and use the `GCS_b37_CHR20_21_REFERENCE` directly?

Reply:
I'm going to keep this because it matches the style of the rest of the test and on the off chance that we want to change this reference path again here.